src/changelog/comment_changes.rs

```diff
@@ -88,7 +88,11 @@ pub struct CommentChanges {
 
     /// The commit to stop at.
     #[arg(long, short = 'S')]
-    stop_at: Option<git2::Oid>,
+    stop_at: Option<Oid>,
+
+    /// The tag to stop at.
+    #[arg(long, short = 'T')]
+    tag: Option<String>,
```

Comment:
This PR adds a new CLI option:  `--tag`.  It will cause the harvesting to stop when the corresponding commit is reached.